tests/subintf_test.py

```diff
@@ -141,6 +147,21 @@ def test_invalid_subintf_creation(self):
         print(result.exit_code, result.output)
         assert result.exit_code != 0
 
+    def test_subintf_creation_on_lag_member(self):
```

Comment:
Inline is it possible to add test to check creation of subintf on vlan members?

Reply:
That's a different functionality check not related to this issue. I recommend you to raise it if it is missed.

Reply:
Agreed. Lets track it separately for negative scenario of subintf creation on vlan member.